src/action/services/useFuncOfStorage.js

```diff
@@ -0,0 +1,44 @@
+import useLocalStorage from "./useLocalStorage.js";
+
+function useFuncOfStorage() {
+  const localStorageHandler = useLocalStorage();
+
+  const getMovieDetailsById = (id) => {
+    return localStorageHandler.readFromLocalStorage(`${id}`);
+  };
+
+  const getObjectByCategory = (category, num) => {
+    const categoryArray = localStorageHandler.readFromLocalStorage(category);
+
+    if (categoryArray === null) {
+      return [];
+    }
+
+    const categoryObjects = categoryArray.split(",").slice(0, num);
+    const categoryArrayWithDetails = [];
+    for (const itemId of categoryObjects) {
+      const itemDetails = localStorageHandler.readFromLocalStorage(itemId);
+
+      if (itemDetails !== null) {
+        categoryArrayWithDetails.push(itemDetails);
+      }
+    }
+
+    return categoryArrayWithDetails;
+  };
+
+  return {
```

Comment:
create the functions instead of defining them in the return statement .

Reply:
dont, everyt function is exported instead of returned

Reply:
i meant done*